LM-1201/99-beta/hello-python.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""DevNet Express - Hello Python!
+
+Running this script will help you verify that your Python environment is setup
+correctly.  It will display information about your Python environment including
+the location of your Python interpreter, where your `pip install ...` packages
+are being installed and more.
+
+Usage:
+    python hello-python.py
```

Comment:
Hi again Chris, yep, I had to run with `python3 hello-python.py` -- should that be indicated here? I also changed